### PR TITLE
Code changed in Introduction/PythonIfElse.py

### PR DESCRIPTION
**Pull request by Utkarsh1504**
I have just done a small code change in Python-IfElse file.

### CODE REVIEW

File: Introduction/PythonIfElse.py [hunk 1]
```diff
@@ -2,24 +2,18 @@
...
+Author    : Utkarsh Mishra
+Created   : 27 Jan 2021
 Problem   : https://www.hackerrank.com/challenges/py-if-else/problem
 '''
 #!/bin/python3
 
 import sys
 
+n = int(input())
+if(n%2==1) or n in range (5,21):
```
Comment (arsho) [inline Introduction/PythonIfElse.py:14]: remove unwanted parenthesis. It can be n%2 == 1

File: Introduction/PythonIfElse.py [hunk 2]
```diff
@@ -2,24 +2,18 @@
...
-nw = 'Not Weird'
-if n % 2 == 1:
-    print(w)
-elif n % 2 == 0 and (n>=2 and n<5):
-    print(nw)
-elif n % 2 == 0 and (n>=6 and n<=20):
-    print(w)
-elif n % 2 == 0 and (n>20):
-    print(nw)    
+//I think this is simpler and easy from the previous.
```
Comment (arsho) [inline Introduction/PythonIfElse.py:19]: remove this line

**Review by arsho — COMMENTED** [review]
Dear Utkarsh,

Thank you for providing a better solution. I have added some feedback. Please address those.

File: Introduction/PythonIfElse.py (hunk 2 above)
Comment (Utkarsh1504) [reply Introduction/PythonIfElse.py]: Thank you for your feedback. I have fixed it now.

**Pull request merged**